Tests/UnitTests/Plugins/DD4hep/CMakeLists.txt

```diff
@@ -41,5 +42,5 @@ if (EXISTS ${DD4hep_DIR}/${CMAKE_INSTALL_LIBDIR})
     endif()
   endforeach()
 else()
-    message(WARNING "DD4hep libraries not found under ${DD4hep_DIR}/${CMAKE_INSTALL_LIBDIR}, DD4hep tests will not be built.")
-endif()
\ No newline at end of file
+    message(WARNING "DD4hep libraries not found under, DD4hep tests will not be built.")
```

Comment:
```suggestion
    message(WARNING "DD4hep libraries not found, DD4hep tests will not be built.")
```

Reply:
Actually I wanted to remove this change from this PR, sorry!